schema/pgsql/schema.sql

```diff
@@ -146,61 +152,67 @@ CREATE TABLE rotation (
     changed_at bigint NOT NULL,
     deleted boolenum NOT NULL DEFAULT 'n',
 
+    CONSTRAINT pk_rotation PRIMARY KEY (id),
+
     -- Each schedule can only have one rotation with a given priority starting at a given date.
     -- Columns schedule_id, priority, first_handoff must be NULLed for deletion via "deleted = 'y'".
-    UNIQUE (schedule_id, priority, first_handoff),
-    CHECK (deleted = 'y' OR priority IS NOT NULL AND first_handoff IS NOT NULL),
+    CONSTRAINT uk_rotation_schedule_schedule_priority_first_handoff UNIQUE (schedule_id, priority, first_handoff),
+    CONSTRAINT ck_rotation_deleted_priority_first_handoff CHECK (deleted = 'y' OR priority IS NOT NULL AND first_handoff IS NOT NULL),
```

Comment:
This is a very similar constraint like `ck_rotation_member_non_deleted_needs_position` and thus should be named similarly: `ck_rotation_non_deleted_needs_priority_first_handoff`.

Reply:
This would probably also better describe what this check is used for, and of course be more consistent with the existing check names.

Reply:
Descriptive names are already being used elsewhere, for example in:
https://github.com/Icinga/icinga-notifications/blob/3fd457278684894214a97d98a4ac1a406748a3a0/schema/pgsql/schema.sql#L202